contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -58,8 +58,7 @@ contract SimpleInterestTermsContract is TermsContract {
     }
 
     modifier onlyThisContract(bytes32 agreementId) {
```

Comment:
Let's change the name to `onlyMappedToThisContract`.  Still confusing, but slightly less confusing :)

Reply:
Done